mypyc/ir/rtypes.py

```diff
@@ -443,28 +443,88 @@ def deserialize(cls, data: JsonDict, ctx: 'DeserMaps') -> 'RTuple':
 )
 
 
+def compute_rtype_size(typ: RType) -> int:
+    """Compute unaligned size of rtype"""
+    platform_size = 4 if IS_32_BIT_PLATFORM else 8
+    if isinstance(typ, RPrimitive):
+        if is_int32_rprimitive(typ):
+            return 4
+        elif is_int64_rprimitive(typ):
+            return 8
+        elif is_none_rprimitive(typ):
+            return 1
+        elif is_bool_rprimitive(typ):
+            return 1
+        else:
+            return platform_size
+    elif isinstance(typ, RTuple):
+        return compute_aligned_offsets_and_size(list(typ.types))[1]
+    elif isinstance(typ, RUnion):
+        return compute_aligned_offsets_and_size(typ.items)[1]
+    elif isinstance(typ, RStruct):
+        return compute_aligned_offsets_and_size(typ.types)[1]
+    elif isinstance(typ, RInstance):
+        return platform_size
+    else:
+        assert False, "invalid rtype for computing size"
+
+
+def compute_aligned_offsets_and_size(types: List[RType]) -> Tuple[List[int], int]:
+    """Compute offsets and total size of a list types after alignment"""
```

Comment:
"list types" -> "list of types"? Make it explicit that the types argument are types of values that are stored sequentially with platform default alignment.